Assets/Scripts/API/ApiMethods.cs

```diff
@@ -342,6 +342,11 @@ private static ReferenceImage _LoadReferenceImage(string location)
         [ApiEndpoint("image.import", "Imports an image given a url or a filename in Media Library\\Images")]
         public static void ImportImage(string location)
         {
+            if (location.StartsWith("http://") || location.StartsWith("https://"))
+            {
+                location = _DownloadMediaFileFromUrl(location, "Images");
```

Comment:
Do you need to delete this at any point after the import is complete?

Reply:
No. Open Brush relies on the file existing locally. Probably should document the behaviour more clearly though.

Reply:
Gotcha, so even though it  "imports", it doesn't duplicate. It's more of a link.